<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>621</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>821</width>
      <height>621</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="minimumSize">
      <size>
       <width>821</width>
       <height>0</height>
      </size>
     </property>
     <widget class="QLabel" name="menurl_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>801</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Wpisz dane</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="username_label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Username_in">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>321</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="password_label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>200</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Password_in">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>250</y>
        <width>321</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="login_button">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>350</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Log in</string>
      </property>
     </widget>
     <widget class="QPushButton" name="register_button">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>430</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sign up</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="text_lebel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="add_friend_button">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Dodaj znajomego</string>
      </property>
     </widget>
     <widget class="QPushButton" name="remove_friend_button">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuń znajomego</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view_friend_button">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Wyświetl znajomych</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logout_button">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>20</y>
        <width>87</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Wyloguj</string>
      </property>
     </widget>
     <widget class="QPushButton" name="group_button">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>140</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Wyślij wiadomość grupową</string>
      </property>
     </widget>
     <widget class="QPushButton" name="priv_button">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>70</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Wyślij wiadomość prywatną</string>
      </property>
     </widget>
     <widget class="QPushButton" name="create_group_button">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>210</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Utwórz grupę</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_lebel_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>320</y>
        <width>351</width>
        <height>211</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
